### PR TITLE
closes #229: Implement config communication

### LINKED ISSUE #229: Communicate important config from server to client
The server needs to send some configuration information to the client during angular's config phase.

This means that something blocking before the document is ready is preferable.

### PR DESCRIPTION
**Pull request by RyanBard**
* Adds a way for the server to send config to the client.
* Pushes google oauth client ID into this config.
* Refactors server config into the same place as "public" client config.
* Updates the unit tests.
* Adds John and KP as admins.

### CODE REVIEW

File: src/main/java/org/hmhb/config/ConfigController.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigController.class);
+
+    private final HttpServletResponse response;
+    private final ConfigService configService;
+    private final ObjectMapper objectMapper;
+
+    /**
+     * An injectable constructor.
+     *
+     * @param response TODO
```
Comment (RyanBard) [inline src/main/java/org/hmhb/config/ConfigController.java]: nope

File: src/main/java/org/hmhb/config/ConfigController.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+     *
+     * @return our single page app's index.html
+     * @throws JsonProcessingException if the {@link ObjectMapper} failed to
+     * serialize the config
+     */
+    @Timed
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/config.js"
+    )
```
Comment (mckoon) [inline src/main/java/org/hmhb/config/ConfigController.java]: Will this be returned as content type "application/javascript"? I can't remember if Spring does this for you with *.js

Comment (RyanBard) [inline src/main/java/org/hmhb/config/ConfigController.java]: Yea it seems to do it for you.  In an earlier iteration of the code, I was trying to specify produces "application/javascript" and it wasn't working.

In the current one (where I don't specify), I had checked and the server was producing the correct media type.

Spring magic I guess.

Comment (RyanBard) [inline src/main/java/org/hmhb/config/ConfigController.java]: (by "it wasn't working", I mean I was getting 406 not acceptable errors returned from the server)

**Pull request merged**